includes/class-wcpdf-admin.php

```diff
@@ -1386,19 +1287,95 @@ public function sort_orders_by_numeric_invoice_number( $query ): void {
 	}
 
 	/**
-     * Determines if the invoice number is numeric.
-     * It evaluates the presence of non-numeric characters in the prefix and suffix of the invoice number.
-     *
+	 * Determines if the invoice number is numeric.
+	 * It evaluates the presence of non-numeric characters in the prefix and suffix of the invoice number.
+	 *
 	 * @return bool
 	 */
-	private function is_invoice_number_numeric() {
+	private function is_invoice_number_numeric(): bool {
 		$invoice_settings = WPO_WCPDF()->settings->get_document_settings( 'invoice' );
 		$is_numeric       = ( empty( $invoice_settings['number_format']['prefix'] ) || ctype_digit( $invoice_settings['number_format']['prefix'] ) ) &&
-		                    ( empty( $invoice_settings['number_format']['suffix'] ) || ctype_digit( $invoice_settings['number_format']['suffix'] ) );
+							( empty( $invoice_settings['number_format']['suffix'] ) || ctype_digit( $invoice_settings['number_format']['suffix'] ) );
 
 		return apply_filters( 'wpo_wcpdf_invoice_number_is_numeric', $is_numeric );
 	}
 
+	/**
+	 * Append ready documents to the heartbeat response.
+	 *
+	 * @param array $response
+	 * @param array $data
+	 *
+	 * @return array
+	 */
+	public function handle_heartbeat_received( array $response, array $data ): array {
+		if ( empty( $data['wpo_pending_documents'] ) || empty( $data['wc-refresh-order-lock'] ) ) {
```

Comment:
This doesn't seem to run for me because in my case I'm not getting the key `wc-refresh-order-lock` but `wp-refresh-post-lock` instead.

Reply:
Also, maybe `wpo_wcpdf_pending_documents` would be more appropriate?